pyi.py

```diff
@@ -273,17 +280,13 @@ def _is_object(node: ast.expr, name: str, *, from_: Container[str]) -> bool:
         where <parent> is a string that can be found within the `from_` collection of
         strings.
 
-    >>> import ast
-    >>> node1 = ast.Name(id="Literal")
-    >>> node2 = ast.Attribute(value=ast.Name(id="typing"), attr="Literal")
-    >>> node3 = ast.Attribute(value=ast.Name(id="typing_extensions"), attr="Literal")
     >>> from functools import partial
     >>> _is_Literal = partial(_is_object, name="Literal", from_=_TYPING_MODULES)
-    >>> _is_Literal(node1)
+    >>> _is_Literal(_ast_node_for("Literal"))
```

Comment:
Do we actually run these doctests in CI?

Reply:
Seems like we do `pyproject.toml:addopts = "--doctest-modules"`.

Reply:
Yup:

https://github.com/PyCQA/flake8-pyi/blob/b7f7e9a7cfab24862ae8bf0f620ad8b84e740d96/pyproject.toml#L19